lib/elasticband/aggregation/nested.rb

```diff
@@ -1,18 +1,24 @@
 module Elasticband
   class Aggregation
```

Comment:
A class that is a namespace, but also a super class which a child class inherits from. Seems complex inheritance stuff :/